src/sentry/constants.py

```diff
@@ -87,7 +87,7 @@ def get_all_languages():
         'security', 'terms', 'from', 'sponsorship', 'for', 'at', 'platforms', 'branding', 'vs',
         'answers', '_admin', 'support', 'contact', 'onboarding', 'ext', 'extension', 'extensions',
         'plugins', 'themonitor', 'settings', 'legal', 'avatar', 'organization-avatar',
-        'project-avatar', 'team-avatar',
+        'project-avatar', 'team-avatar', 'careers',
```

Comment:
Out of curiosity, why are these in `sentry` and not `getsentry`?

Reply:
It's just where they are. There's no reason they can't be moved into getsentry. We'd have to have an overlapping list since there are ones that need to be restricted for all of sentry, then our specific getsentry ones.